src/shipping-zones/dto/create-shipping-zone.dto.ts

```diff
@@ -1,5 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger'
+
 export class CreateShippingZoneDto {
+
+  @ApiProperty()
+  name: string;
+
+  @ApiProperty()
+  countries: string[];
```

Comment:
This should be counties (νομοί) since it represents the various counties in a shipping zone, for example a zone might include the counties of "Αχαΐας", "Ηλείας" and "Αιτωλοακαρνανίας".